minions/async/src/test/java/io/github/microcks/minion/async/AsyncAPITestManagerIT.java

```diff
@@ -166,4 +166,94 @@ private static void sendTextMessagesOnTopic(int number) {
+         public List<Service> listServices(String authorization, int page, int size) {
+            return null;
+         }
+
+         @Override
+         public ServiceView getService(String authorization, String serviceId, boolean messages) {
+            return null;
+         }
+
+         @Override
+         public List<Resource> getResources(String serviceId) {
+            if ("d3d5a3ed-13bf-493f-a06d-bf93392f420b".equals(serviceId)) {
+               Resource resource = new Resource();
+               resource.setId("578497d2-2695-4aff-b7c6-ff7b9a373aa9");
+               resource.setType(ResourceType.ASYNC_API_SPEC);
+               resource.setContent(asyncAPIContent);
+               return List.of(resource);
+            }
+            return Collections.emptyList();
+         }
+
+         @Override
+         public TestCaseResult reportTestCaseResult(String testResultId, TestCaseReturnDTO testCaseReturn) {
+            reportedTestCases.put(testResultId, testCaseReturn);
+            return null;
+         }
+      };
+      SchemaRegistry schemaRegistry = new SchemaRegistry(microcksAPIConnector);
+      schemaRegistry.updateRegistryForService("d3d5a3ed-13bf-493f-a06d-bf93392f420b");
+
+      AsyncTestSpecification testSpecification = new AsyncTestSpecification();
+      testSpecification.setServiceId("d3d5a3ed-13bf-493f-a06d-bf93392f420b");
+      testSpecification.setEndpointUrl(
+            "kafka://%s/%s".formatted(kafkaContainer.getBootstrapServers().replace("PLAINTEXT://", ""), TOPIC_NAME));
+      testSpecification.setRunnerType(TestRunnerType.ASYNC_API_SCHEMA);
+      testSpecification.setAsyncAPISpec(asyncAPIContent);
+      testSpecification.setOperationName("SEND sendEchoMessage");
+      testSpecification.setTimeoutMS(2200L);
+      testSpecification.setTestResultId("test-result-id");
+
+      AsyncAPITestManager manager = new AsyncAPITestManager(microcksAPIConnector, schemaRegistry);
+      manager.microcksUrl = "http://microcks:8080";
+
+      // Act.
+      manager.launchTest(testSpecification);
+
+      // Wait a bit so that consumption task has actually started.
+      await().during(1250, TimeUnit.MILLISECONDS).until(() -> true);
+      Properties props = new Properties();
+      props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, kafkaContainer.getBootstrapServers());
+      props.put(ProducerConfig.CLIENT_ID_CONFIG, "microcks-async-minion-str-producer");
+      props.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
+      props.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
+
+      KafkaProducer<String, String> producer = new KafkaProducer<>(props);
+
+      for (int i = 0; i < 5; i++) {
+         ProducerRecord<String, String> kafkaRecord = new ProducerRecord<>(TOPIC_NAME,
+               String.valueOf(System.currentTimeMillis()), "test");
+         producer.send(kafkaRecord);
+      }
+      producer.flush();
+      producer.close();
```

Comment:
Isn't it a duplication of the `sendTextMessagesOnTopic()` private method? Not really the same as you're just sending `test` string but... I don't get the point.